CHANGELOG.md

```diff
@@ -670,3 +672,5 @@ await myHelper(context);
 [0.12.0]: https://github.com/coda/packs-sdk/compare/v0.11.0...v0.12.0
 [0.11.0]: https://github.com/coda/packs-sdk/compare/v0.9.0...v0.11.0
 [0.9.0]: https://github.com/coda/packs-sdk/compare/v0.8.2...v0.9.0
+
+[1.7.2]: https://github.com/coda/packs-sdk/compare/v1.7.1...v1.7.2
```

Comment:
The ordering of these never seems to work for me. I'm going to manually fix it again